### DISCUSSION TITLE
Heat Map Chart - Row Wise Heat Mapping

### DISCUSSION THREAD

**Discussion by tmg-haroldbright** [nhn/tui.chart#704] [Ideas]
At present the Heat Map Chart takes in account the complete dataset and plots the headmap over it. However is it possible to have it applied for each row or column data only as a group. Something like which we can do in MSExcel.

**Comment by jwlee1108** [comment]
@tmg-haroldbright 

Hi, I'm sorry for the late response. Thank you for your ideas. Can you explain it a little more specifically?
